services/storage/src/simcore_service_storage/cli.py

```diff
@@ -26,7 +26,8 @@ def run():
     logging.basicConfig(level=settings_obj.log_level)
     logging.root.setLevel(settings_obj.log_level)
     config_all_loggers(
-        log_format_local_dev_enabled=settings_obj.STORAGE_LOG_FORMAT_LOCAL_DEV_ENABLED
+        log_format_local_dev_enabled=settings_obj.STORAGE_LOG_FORMAT_LOCAL_DEV_ENABLED,
+        logger_filter_mapping=settings_obj.STORAGE_LOG_FILTER_MAPPING,
```

Comment:
on the line below that is what I meant with the NOISY_LOGGERS.

Reply:
Yes, we can move to similar logic and have an ENV var for those.